spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -16,30 +16,33 @@ This standard document specifies packet data structure, state machine handling l
-- `Interchain Account Owner`: An account on the controller chain. Every interchain account on a host chain has a respective owner account on the controller chain 
-- `Controller Chain`: The chain registering and controlling an account on a host chain. The controller chain sends IBC packets to the host chain to control the account
-- `Host Chain`: The chain where the interchain account is registered. The host chain listens for IBC packets from a controller chain which should contain instructions (e.g. cosmos SDK messages) that the interchain account will execute
+- `Interchain Account`: An account on a host chain. An interchain account has all the capabilities of a normal account. However, rather than signing transactions with a private key, a controller chain will send IBC packets to the host chain which signals what transactions the interchain account should execute. 
+- `Interchain Account Owner`: An account on the controller chain. Every interchain account on a host chain has a respective owner account on the controller chain. 
+- `Controller Chain`: The chain registering and controlling an account on a host chain. The controller chain sends IBC packets to the host chain to control the account.
+- `Host Chain`: The chain where the interchain account is registered. The host chain listens for IBC packets from a controller chain which should contain instructions (e.g. cosmos SDK messages) that the interchain account will execute.
 
 The IBC handler interface & IBC relayer module interface are as defined in [ICS 25](../ics-025-handler-interface) and [ICS 26](../ics-026-routing-module), respectively.
 
 ### Desired Properties
 
 - Permissionless 
-- The ordering of transactions sent to an interchain account on a host chain must be maintained. Transactions should be executed by an interchain account in the order in which they are sent by the controller chain
-- If a channel closes, the controller chain must be able to regain access to registered interchain accounts by simply opening a new channel
-- Each interchain account is owned by a single account on the controller chain. Only the owner account on the controller chain is authorized to control the interchain account
-- The controller chain must store the account address of the interchain account
+- Fault tolerance: A controller chain must not be able to control accounts registered by other controller chains. For example, in the case of a fork attack on a controller chain, only the interchain accounts registered by the forked chain will be vulnerable.
+- The ordering of transactions sent to an interchain account on a host chain must be maintained. Transactions should be executed by an interchain account in the order in which they are sent by the controller chain.
+- If a channel closes, the controller chain must be able to regain access to registered interchain accounts by simply opening a new channel.
```

Comment:
Really? Why, and how is this made safe?

Reply:
I see that you tie accounts to the connection identifiers. I think this is fine, but I don't think it really gives any additional "fault liveness recovery", which I presume is what you want. If clients expire a new connection will need to be created in addition to new channels.

Reply:
So we are considering two possibilities for the channel to close:

1. Timeout on ORDERED channel

In this case, the user can simply open a new channel on the same connection with the same port and that new channel will be able to control the original interchain account.

2. Underlying client expires or gets frozen for misbehaviour.

Here ICA does not have any in-protocol method for recovery, you are right. However, the underyling layers may have their own recovery mechanisms such that we don't have to create new connections/channels.
In fact, this is exactly the case in the ibc-go implementation. If a client expires or is frozen, then governance can come in and revive the client such that the connections/channels can resume working without having to create a whole new one.
So here, we can rely on governance to hopefully resolve the issue.
It's possible of course that governance does not choose to do this, in which case, there is nothing we can do.

Perhaps this reasoning should be written down succinctly @seantking

Reply:
Right, OK, so the case that we're really concerned about here is (1), since that is the case where using connection identifiers allows for recovery in this case of timeouts of packets _but not_ the client. That makes sense to me. However, the semantics seem identical to (and more complicated than) an ordered channel which just doesn't close on timeouts, so maybe we should aim to add support for that instead and avoid the unnecessary extra handshake?

AFAIK (2) doesn't benefit from the possibility of using multiple channels so isn't relevant to this design question.